util.go

```diff
@@ -43,3 +45,36 @@ func IsKeyInDomain(key, start, end []byte) bool {
 	}
 	return true
 }
+
+func FileExists(filePath string) bool {
```

Comment:
same here. should probably be unexported

Reply:
used in fsdb.go